### PR TITLE
Bump selenium-webdriver, implement aspect sorting cucumber step with …

### PR DESCRIPTION
**Pull request by jhass**
…selenium API

Partial backport of a2d894cb391c0d2afbe6a22187990eb5a0ede8b8

Let's see what Travis thinks.

### CODE REVIEW

File: features/step_definitions/aspects_steps.rb [hunk 1]
```diff
@@ -88,24 +88,21 @@ def aspect_dropdown_visible?
...
+  browser = aspect.base.driver.browser
+  mouse = browser.mouse
+  native_aspect = aspect.base.native
+  native_target = target.base.native
+  mouse.down native_aspect
+  mouse.move_to native_target, native_target.size.width/2, 0
+  sleep 1
+  mouse.up
+  expect(page).to have_no_css "#aspect_nav .ui-sortable.syncing"
+ end
```
Comment (diaspora-code-review) [inline features/step_definitions/aspects_steps.rb]: `end` at 105, 1 is not aligned with `When /^I drag "([^"]*)" (up|down)$/ do |aspect_name, direction|` at 91, 0

File: features/step_definitions/aspects_steps.rb [hunk 2]
```diff
@@ -88,24 +88,21 @@ def aspect_dropdown_visible?
...
-          $.getScript("/assets/jquery-simulate-ext/src/jquery.simulate.drag-n-drop.js", drag);
-        });
-      });
-    }
-    if (!$.simulate) { loadScripts(); } else { drag(); }
-  }
-  expect(find("#aspect_nav")).to have_css ".synced"
-end
+When /^I drag "([^"]*)" (up|down)$/ do |aspect_name, direction|
+  aspect_id = @me.aspects.where(name: aspect_name).first.id
```
Comment (diaspora-code-review) [inline features/step_definitions/aspects_steps.rb]: Use 2 (not 1) spaces for indentation.

File: features/step_definitions/aspects_steps.rb [hunk 3]
```diff
@@ -88,24 +88,21 @@ def aspect_dropdown_visible?
...
+  aspect = find(:xpath, "//div[@id='aspect_nav']/ul/li[@data-aspect-id='#{aspect_id}']")
+  target = direction == "up" ? aspect.all(:xpath, "./preceding-sibling::li").last :
```
Comment (diaspora-code-review) [inline features/step_definitions/aspects_steps.rb]: Avoid multi-line ?: (the ternary operator); use `if`/`unless` instead.

File: features/step_definitions/aspects_steps.rb [hunk 4]
```diff
@@ -88,24 +88,21 @@ def aspect_dropdown_visible?
...
+                               aspect.all(:xpath, "./following-sibling::li").first
+  browser = aspect.base.driver.browser
+  mouse = browser.mouse
+  native_aspect = aspect.base.native
+  native_target = target.base.native
+  mouse.down native_aspect
+  mouse.move_to native_target, native_target.size.width/2, 0
```
Comment (diaspora-code-review) [inline features/step_definitions/aspects_steps.rb]: Surrounding space missing for operator `/`.

**Comment by jhass** [comment]
Derp, forgot to change the target, sorry.

**Pull request closed without merging**